.woodpecker.yml

```diff
@@ -38,4 +38,4 @@ pipeline:
   dead_links:
     image: becheran/mlc:latest
     commands:
```

Comment:
What's your rationale for this, I don't care one way or the other and dropping the commit is trivial, but why keep it disabled?

Can I instead amend the commit to purge the CI that you're not using?

This is effectively 'dead code' prior to my change and I don't understand the value of treating CI configuration that you're _not_ using as 'live code' when it can just as easily (and if not in use frankly should) reside in the commit history.

Reply:
We've had a lot of failed builds because of mlc (external links go down a lot).